lib/exception_handling/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module ExceptionHandling
-  VERSION = '2.4.0'
+  VERSION = '2.5.0'
```

Comment:
Not a patch? This does not change any of the functionality of the gem.

Reply:
Ok, I'll make it a patch.

Reply:
Reset to 2.4.1.